### PR TITLE
change date format and listItem font size

### LINKED ISSUE #1074: [sanity] Change date format
[Discord thread feedback](https://discord.com/channels/841390338324824096/1099956633376923699)

DOMAIN = both

> Could we maybe standardise the date format on the Resources pages to one that's more intuitive? The 'MM/DD/YYYY' format is confusing for a lot of people. I would suggest 'Month DD, YYYY' (e.g. April 11, 2023), as that's the format we use in social copy, blogs, etc., but if it's got to be numerals only then 'DD/MM/YYYY' would definitely be more user friendly.

image

Seems to fine on the article pages themselves, so this change would apply to https://www.carbonmark.com/resources and the dates on the article cards.

### PR DESCRIPTION
**Pull request by PaulineAnnBar**
## Related Ticket

Resolves #1074 #1044 

## Checklist

- [x] I have run `npm run build-all` without errors
- [x] I have formatted JS and TS files with `npm run format-all`

### CODE REVIEW

**Comment by PaulineAnnBar** [comment]
@jabby09 here is the PR.

File: package-lock.json [hunk 1]
```diff
@@ -28576,6 +28576,7 @@
     },
     "node_modules/tap/node_modules/@ampproject/remapping": {
       "version": "2.1.2",
+      "dev": true,
```
Comment (ladytrekker) [inline package-lock.json:28579]: Please remove package.lock from this PR

File: examples/nextjs-typescript-integration/abis/ERC20.json [hunk 2]
```diff
@@ -185,4 +185,4 @@
       "type": "function"
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (ladytrekker) [inline examples/nextjs-typescript-integration/abis/ERC20.json:188]: Can you remove all changes from files in the /examples/ folder from this PR?

I know, they got changed when running `npm run format-all` 
Still, these changes belong to another PR. Thanks

File: carbonmark/components/pages/Blog/Post/index.tsx [hunk 3]
```diff
@@ -40,9 +40,11 @@ export const PostPage = (props: PostProps) => {
...
   }
 
-  const publishedDate = `Published ${new Date(
-    props.post.publishedAt
-  ).toDateString()}`;
+  const publishedDate = `Published ${new Date(props.post.publishedAt)
+    .toDateString()
+    .split(" ")
+    .slice(1)
+    .join(" ")}`;
```
Comment (ladytrekker) [inline carbonmark/components/pages/Blog/Post/index.tsx:47]: As multiple languages need to be supported and Dates are formatted differently depending on each language
=> let's rather use `Intl.DateFormat` (see [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat))

So in this case this would be:

```TS

import { useRouter } from "next/router";

...
const { locale } = useRouter();

const publishedDate = new Date(props.post.publishedAt);

const formattedDate = new Intl.DateTimeFormat(locale, {
      day: "numeric",
      month: "long",
      year: "numeric",
   }).format(publishedDate);
```
Results for english => May 30, 2023
Results for german => 30. Mai 2023
Results for Hindi => 30 मई 2023

... and so on ...

File: carbonmark/components/pages/Resources/BlogPostCard/index.tsx [hunk 4]
```diff
@@ -10,7 +10,11 @@ interface BlogPostCardProps {
 }
 
 export const BlogPostCard = (props: BlogPostCardProps) => {
-  const date = new Date(props.post.publishedAt).toLocaleDateString("en");
+  const date = new Date(props.post.publishedAt)
+    .toDateString()
+    .split(" ")
+    .slice(1)
+    .join(" ");
```
Comment (ladytrekker) [inline carbonmark/components/pages/Resources/BlogPostCard/index.tsx:17]: Same here, see comment above

File: site/components/PodcastCard/index.tsx [hunk 5]
```diff
@@ -12,7 +12,11 @@ interface CardProps {
 }
 
 export function PodcastCard(props: CardProps) {
-  const date = new Date(props.podcast.publishedAt).toLocaleDateString("en");
+  const date = new Date(props.podcast.publishedAt)
+    .toDateString()
+    .split(" ")
+    .slice(1)
+    .join(" ");
```
Comment (ladytrekker) [inline site/components/PodcastCard/index.tsx:19]: Same here

File: site/components/pages/Blog/Post/index.tsx [hunk 6]
```diff
@@ -34,9 +34,11 @@ export const PostPage = (props: PostProps) => {
...
   }
 
-  const publishedDate = `Published ${new Date(
-    props.post.publishedAt
-  ).toDateString()}`;
+  const publishedDate = `Published ${new Date(props.post.publishedAt)
+    .toDateString()
+    .split(" ")
+    .slice(1)
+    .join(" ")}`;
```
Comment (ladytrekker) [inline site/components/pages/Blog/Post/index.tsx:41]: Same

File: site/components/pages/Resources/FeaturedArticles/Article.tsx [hunk 7]
```diff
@@ -16,11 +15,11 @@ type Props = {
...
-  const { locale } = useRouter();
-
-  const formattedDate = new Date(props.article.publishedAt).toLocaleDateString(
-    locale
-  );
+  const formattedDate = new Date(props.article.publishedAt)
+    .toDateString()
+    .split(" ")
+    .slice(1)
+    .join(" ");
```
Comment (ladytrekker) [inline site/components/pages/Resources/FeaturedArticles/Article.tsx:22]: Same

**Review by ladytrekker — CHANGES_REQUESTED** [review]
Thanks @PaulineAnnBar ⭐ 

Some minor changes requested below.

**Review by jabby09 — APPROVED** [review]
Looks good for both on the front end.

File: carbonmark/components/pages/Blog/Post/index.tsx (hunk 3 above)
Comment (PaulineAnnBar) [reply carbonmark/components/pages/Blog/Post/index.tsx:47]: Of course, makes sense! Done

**Review by ladytrekker — APPROVED** [review]
Nice! Thanks a bunch 💯
